glyphs-reader/src/font.rs

```diff
@@ -92,6 +92,7 @@ struct RawFont {
 pub struct RawFeature {
     pub automatic: Option<i64>,
     pub name: Option<String>,
+    pub tag: Option<String>,
```

Comment:
Is *this* always an OpenType tag?

Reply:
I can't think what else it would be but I don't have strong documentation to point to.